rotkehlchen/assets/utils.py

```diff
@@ -122,6 +129,20 @@ def _edit_token_and_clean_cache(
         GlobalDBHandler().edit_evm_token(evm_token)
 
 
+def test_if_is_spam_token(symbol: Optional[str]) -> bool:
+    """Makes basic checks to test if a token could be spam or not"""
+    if symbol is None:
+        return False
+    if (
```

Comment:
yes but in the name I haven't seen such good patterns. They usually add this to the symbol because is what is rendered in almost every platform first and bigger